packages/opentelemetry-tracing/src/utility.ts

```diff
@@ -21,14 +21,19 @@ import {
   DEFAULT_MAX_LINKS_PER_SPAN,
 } from './config';
 import { TracerConfig } from './types';
+import { ProbabilitySampler, getEnv } from '@opentelemetry/core';
 
 /**
  * Function to merge Default configuration (as specified in './config') with
  * user provided configurations.
  */
 export function mergeConfig(userConfig: TracerConfig) {
```

Comment:
missing params

Reply:
Should we really document the params since their types are documented by the typings anyway ?

Reply:
I don't think it's needed. Most editor integrations look at the `d.ts` files for param docs anyways.